.changeset/serious-windows-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': minor
```

Comment:
All of these are functionally not changing anything that can be seen externally, so they can be demoted to just `patch` level. And it's fine to make a single changeset for that reason, with a text along the lines `Internal refactor to avoid usage of deprecated symbols` or something like that.

Reply:
Ok I'll be fixing that!